gpio_local.py

```diff
@@ -0,0 +1,29 @@
+from pin import Pin, MAPPINGS
+
+PIN_NUMBER = input(
+    'Please enter one of these pin numbers for the GPIO '
+    f'you would like to use: {list(MAPPINGS.keys())}'
+)
+
+if PIN_NUMBER not in MAPPINGS.keys():
+    print(
+        'Your *pin* number is not found, can you make sure '
+        'you are using the PIN number not the GPIO?'
+    )
+
+if PIN_NUMBER in MAPPINGS.keys():
+    pin = Pin(PIN_NUMBER)
+    pin.configure()
+    print(
+        'Please enter ON or OFF to turn the pin on or off. '
+        'When finished, enter CLEAR to clear the pin.'
+    )
+    while True:
+        REQUEST = input('Enter ON, OFF, or CLEAR: ')
+        if REQUEST == 'CLEAR':
```

Comment:
So `CLEAR` is an alias for `quit`/`exit`? If so, would it make sense to use a command named this way rather than `clear`?

Reply:
Sounds good, I'll handle it with a finally, technically I think if multiple Control + C prompts fire (the user is impatient) then the finally wont execute but I think it should clean up pretty quick.